README.md

```diff
@@ -1 +1,9 @@
+[![Build Status](https://travis-ci.org/sul-dlss/trusted_checksum_repository.svg?branch=master)](https://travis-ci.org/sul-dlss/trusted_checksum_repository)
+[![Coverage Status](https://coveralls.io/repos/github/sul-dlss/trusted_checksum_repository/badge.svg)](https://coveralls.io/github/sul-dlss/trusted_checksum_repository)
+[![Dependency Status](https://gemnasium.com/badges/github.com/sul-dlss/trusted_checksum_repository.svg)](https://gemnasium.com/github.com/sul-dlss/trusted_checksum_repository)
+[![GitHub version](https://badge.fury.io/gh/sul-dlss%2Ftrusted_checksum_repository.svg)](https://badge.fury.io/gh/sul-dlss%2Ftrusted_checksum_repository)
+
+
 # README
+
+Trusted Checksum Repository is a Rails app for independently storing preservation object checksums. The [Preservation Catalog](https://github.com/sul-dlss/preservation_core_catalog) uses the TCR to perform checksum validation on preserved objects.
```

Comment:
I'm not actually sure the PC will be calling the TCR directly;  that's what I was objecting to earlier.